presto-common/src/main/java/com/facebook/presto/common/block/AbstractArrayBlock.java

```diff
@@ -30,7 +30,7 @@
 public abstract class AbstractArrayBlock
         implements Block
 {
-    protected abstract Block getRawElementBlock();
+    public abstract Block getRawElementBlock();
```

Comment:
Take a look at `ColumnarArray` and `ColumnarRow`. That may be helpful to make the modifiers untouched in presto-common